schema.sql

```diff
@@ -3,15 +3,15 @@ create database bandland;
 \connect bandland;
 
 CREATE TABLE albums(
-  id serial primary key,
+  id int primary key,
   name varchar(100) not null,
   artist varchar(100) not null,
   image varchar(500) not null,
   tags varchar(100) not null,
   description varchar(1000) not null
 );
 
-\copy albums(name, artist, image, tags, description) FROM 'db/data.csv' DELIMITER ',' CSV HEADER;
+\copy albums(id, name, artist, image, tags, description) FROM 'db/data.csv' DELIMITER ',' CSV HEADER;
```

Comment:
So for my COPY command I used the following syntax:
COPY bandland_comments.album_comments (id, albumId, username, comment, profileImg) from '/Users/cms287/Documents/hrr37/SDC/sdc-johnson-cody/commentsCassandra.csv' with HEADER = TRUE;

I used the full file path because I was in my cqlsh which I got into from root directory. I didn't specific a delimiter because it should default to ' , '. Maybe instead of 'CSV HEADER' you can write 'WITH HEADER = TRUE'.

You might also consider having more explicit syntax for establish the connecting to the Cassandra but I'm not  entirely sure since even with seeding Postgres I didn't use .sql file for seeding. 

Sorry I can't offer better suggestions. I'm sure Druthi can better suggestions as took a similar approach to what you have above. 

If you decided to seed from the command line I can walk you through exactly what I did.